### PR TITLE
git-clone: update Indonesian translation

### PR DESCRIPTION
**Pull request by raaynaldo**
- [X] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [X] The page(s) have at most 8 examples.
- [X] The page description(s) have links to documentation or a homepage.
- [X] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.id/common/git-clone.md [hunk 1]
```diff
@@ -1,24 +1,36 @@
...
 
-- Mengkloning repositori yang ada:
+- Klon repositori yang ada:
 
 `git clone {{lokasi_repositori_remote}}`
 
-- Mengkloning reposiori yang ada dan submodulenya:
+- Klon repositori yang ada ke direktori tertentu:
+
+`git clone {{lokasi_repositori_remote}} {{alamat/ke/direktori}}`
```
Comment (kbdharun) [inline pages.id/common/git-clone.md:12]:
```suggestion
`git clone {{lokasi_repositori_remote}} {{jalan/menuju/direktori}}`
```

File: pages.id/common/git-clone.md [hunk 2]
```diff
@@ -1,24 +1,36 @@
...
+
+- Klon repositori yang ada dan submodulnya:
 
 `git clone --recursive {{lokasi_repositori_remote}}`
 
-- Mengkloning repositori lokal:
+- Klon repositori lokal:
 
-`git clone -l {{alamat/ke/repository/lokal}}`
+`git clone -l {{alamat/ke/repositori/lokal}}`
```
Comment (kbdharun) [inline pages.id/common/git-clone.md:20]:
```suggestion
`git clone -l {{jalan/menuju/repositori/lokal}}`
```

**Review by kbdharun — COMMENTED** [review]
The translation LGTM, I have some suggestions for paths. [Reference](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/common-arguments.md)

**Comment by raaynaldo** [comment]
Comments addressed @kbdharun Thank you!

**Review by kbdharun — APPROVED** [review]
LGTM

**Comment by raaynaldo** [comment]
Could you review this one as well @reinhart1010

**Comment by raaynaldo** [comment]
Hi @reinhart1010 , could you review this when you have a chance?

**Comment by raaynaldo** [comment]
Hi @reinhart1010 or @navarroaxel  , could you review this when you have a chance?

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**